vendor/bat-native-ledger/src/bignum.h

```diff
@@ -8,15 +8,7 @@
 
 #include <string>
 
-extern "C" {
```

Comment:
what are changes in this file?

Reply:
bignum fix request per @bridiver

Reply:
let's move this in a separate PR as I don't see how this is related to url's

Reply:
Done